pages/data-design/groups.mdx

```diff
@@ -0,0 +1,132 @@
+---
+layout: 'guide'
+title: 'Intro to Group analytics'
+---
+
+import Link from '../../components/Link';
+
+When doing product analytics you may find yourself in a position when you need to analyze behavior of a group of users, or you need to group
+different actions together into a group.
+
+There are two main ways of group analytics that we are going to look at in this article:
+
+- Grouping users
+- Grouping actions
+
+#### Groups concepts
+
+There are two main concepts when working with group analytics. In Avo they are called “Group Types” and “Group Ids”. See reference table below for what “Group Types” and “Group IDs” are called in other platforms.
+
+Group types are the types of groups you manage, for example companies, branches, chats and so on.
+
+Group ids are identifiers of the individual groups of those group types, for example in the “Company” group type you’d have group ids as names of the companies, like “Alibaba”, “Moderna”, “Uber”.
+
+|                      |            |            |
+| -------------------- | ---------- | ---------- |
+| Concepts in Avo      | Group Type | Group Id   |
+| Concept in Amplitude | Group Type | Group Name |
+| Concept in Mixpanel  | Group Key  | Group Id   |
+| Concept in Segment   | Group Id   | Group Name |
+
+## Grouping users
+
+You can associate users with groups. Then that user profile as well as every action user makes will be attributed to the associated group.
+
+This is useful for example for B2B products, when each individual user of the product belongs to a company, and every action of that user should be attributed to that one company.
+
+See further use cases and which analytics platforms support grouping users below in <Link passHref href="/data-design/groups#associate-user-with-group"><a>Associate user with group</a></Link>.
+
+## Grouping actions
+
+You can associate events with groups without associating the user with a group. This is useful if you have product concepts where each user can perform actions in different groups.
+
+For example if you have a chat app and want to analyze the performance of different channels.
+
+See further use cases and which analytics platforms support grouping events below in <Link passHref href="/data-design/groups#associate-event-with-group"><a>Associate event with group</a></Link>.
+
+## Managing Group Analytics
+
+### Associate user with group
+
+**Functionality**: This action will associate this user with a specific group ID in your downstream data.
+This is supported by Avo, Segment, Mixpanel and Amplitude, and additionally you can use this method for your data warehouse via custom destinations in Avo.
+
+[//]: # 'TODO [TO CONFIRM] Consequences: All future [todo confirm this applies to Mixp and Segm as well] events triggered with this user ID, will be available for group analyses such as calculating metrics based on unique groups.'
+
+**Consequences**: All events triggered with this user ID will be available for group analyses. See
+
+<a
```

Comment:
looks like something's broken here 👀 

![image](https://user-images.githubusercontent.com/2991926/128448973-287e751c-a19c-4a46-af66-beceac5826d2.png)

Reply:
This is because we have a TODO comment in between, and comments in markdown are a bit of a hack, so they require  a new line. Will rearrange it

Reply:
Ah wait, that's not the case, what problem do you mean here?

Reply:
looks like this is resolved :)

![image](https://user-images.githubusercontent.com/2991926/128872308-8b26e4ad-a050-4b82-b0f7-9451ac6d3805.png)